### ISSUE TITLE
Default Package Icons do not scale well at high resolution/DPI

### ISSUE THREAD

**Issue by mishra14** [NuGet/Home#5937]
## Details about Problem

NuGet product used (NuGet.exe | VS UI | Package Manager Console | dotnet.exe): VS UI

NuGet version (x.x.x.xxx): 4.4.4475

VS version (if appropriate): 15.4 Preview 3 | 26923.0.d15rel

## Detailed repro steps so we can see the same problem

1. Change the resolution to 1560 x 1440 or greater.

2. Change the scaling to 175%.

3. Launch VS and create new project.

4. Launch PM UI and search for nugetvalidator or any other package with default icon.
...

## Expected
The default icon should be rendered without aliasing.

## Actual
The default and refresh icons do not have smooth edges - 

![capture](https://user-images.githubusercontent.com/10507120/30837109-70a1fe42-a217-11e7-961d-3cafde47fb59.PNG)
![capture2](https://user-images.githubusercontent.com/10507120/30837108-70a1a37a-a217-11e7-82b5-6b74d90b1019.PNG)

**Comment by donnie-msft** [comment]
Still relevant in 16.x. Designer noticed the refresh icon does not scale, nor match VS Theme.

Evaluate all resource files here: NuGet.PackageManagement.UI\Resources\

Refactoring is needed to move from resource files to 
https://docs.microsoft.com/en-us/visualstudio/extensibility/image-service-and-catalog?view=vs-2019
